src/atomizer.js

```diff
@@ -235,7 +235,15 @@ module.exports = {
             minify: false,
             keepCamelCase: false,
             extCSS: '.css',
-            banner: ''
+            banner: '',
+            namespace: '#atomic',
+            rtl: false,
+            // TODO: Verify these are good defaults
```

Comment:
We might need to change how we deal with breakpoints since @thierryk mentioned the need for more complex use of media-queries. For now, I think it's fine to have defaults breakpoints.